cider/tests/functionality/DateTypeQueryTest.cpp

```diff
@@ -293,6 +293,7 @@ TEST_F(TimeTypeQueryTest, MultiTimeTypeTest) {
 
 int main(int argc, char** argv) {
   testing::InitGoogleTest(&argc, argv);
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
```

Comment:
What's this new added line used for?

Reply:
enable parse gflags, in order to use new data format by set 'is_using_arrow_format_'